md/widgets.md

```diff
@@ -53,6 +53,21 @@ Each tab is a container widget.
 Use the **form container** to enable form validation for input widgets. The form container also exposes a local **$form** which is the AngularJS form object associated to the form container. In the Bonita Studio, if you define a contract for the task or process and click on the UI Designer icon, the generated form already contains a form container to embed default input widgets to collect the information that need to be passed to the contract to be verified. You can then define the validation you need on the input.  
 For example, you can bind the button's disabled property to _$form.$invalid_ to prevent user from clicking the button, until the form inputs are valid.
 
+### <a name="modal-widget"></a> Modal container widget
+
+Use the **modal container** to create a pop-up window. This modal container can be opened with a button by choosing `Open Modal` in the button's "Actions" property. 
+Choose a name for the `modal ID` property so you can define what modal container the button will display.
+This ID must be unique for a given page. 
+Any button located inside a modal window can close the modal on success, during any action, when choosing `close modal` in the Action property.
```

Comment:
Between "open modal" and "close modal", you must repeat the same pattern of capitalized and non-capitalized letters than on the button's actions in the UI Designer: "Open modal" and "Close modal" could be good, but please check how it is written in the UI Designer.